src/order/conditionally_complete_lattice.lean

```diff
@@ -36,8 +36,9 @@ variables {α : Type u} {β : Type v} {ι : Type w}
 section preorder
 variables [preorder α] [preorder β] {s t : set α} {a b : α}
 
-/-Sets bounded above and bounded below.-/
+/-- A set is bounded above. -/
```

Comment:
This sounds to me like a docstring on a theorem that says that every set is bounded above.
```suggestion
/-- A set is bounded above if there exists an upper bound. -/
```